src/TheSpace/Property.sol

```diff
@@ -1,86 +1,113 @@
 //SPDX-License-Identifier: Apache-2.0
-pragma solidity ^0.8.0;
+pragma solidity ^0.8.11;
 
 import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
 
 /**
  * @dev Harberger property, constantly in auction by allowing market contract to transfer token.
  */
 contract Property is ERC721Enumerable {
```

Comment:
any reason why `ERC721Enumerable` instead of `ERC721`?

Reply:
Frontend needs a way to list all tokens owned by a particular address, looks like `ERC721Enumerable` is the easiest way.

Reply:
Minting on ERC721Enumerable needs more gas than basic ERC721, if we really need it, maybe we should do some optimization.

refs:
* https://twitter.com/jonathansnow/status/1460297402783576066
* https://shiny.mirror.xyz/OUampBbIz9ebEicfGnQf5At_ReMHlZy0tB4glb9xQ0E

Reply:
Also, maybe we should try some gas-optimized standard token libraries like https://github.com/Rari-Capital/solmate